src/interfaces/posts.ts

```diff
@@ -135,6 +135,13 @@ export enum BlogType {
     Blog = 'blog',
 }
 
+export const blogType = (frontmatter: FrontMatter): BlogType =>
```

Comment:
I feel like some of the naming in this file can be improved for readability. Since this is a function can `blogType` change to a verb, like `findBlogType`/`determineBlogType`?

Reply:
I totally agree! The whole blog infrastructure needs a bit of an overhaul with some standardization with naming conventions and file placement. I didn't want to spend too much time on it just yet cause it needs some time to give it a lot more love. 🤓

I like the `get` convention and also your suggestion about `postType`. Let's combine to make this `getPostType`. How do you feel about that?